src/Continuous_Learning/continuous-learning-train-and-test.sh

```diff
@@ -0,0 +1,60 @@
+#! /bin/bash
+
+set -e
+set -x
+
+export PATH=$PATH:"$HOME/defects4j/framework/bin"
+
+OpenNMT_py="$HOME/OpenNMT-py"
+CONTINUOUS_LEARNING_PATH="$HOME/sequencer/src/Continuous_Learning"
+DEFECTS4J_EXPERMENT_PATH="$HOME/sequencer/src/Defects4J_Experiment"
+
+VOCABULARY="$HOME/sequencer/results/Golden/vocab.txt"
```

Comment:
I think it is better to update the vocab file.

For example if you are training a new model, the vocabulary can differ a lot to the golden model.